src/components/afm/AreaChart.ts

```diff
@@ -0,0 +1,25 @@
+import { AFM } from '@gooddata/typings';
+import {
+    dataSourceProvider,
+    IDataSourceProviderProps
+} from './DataSourceProvider';
+
+export {
+    IDataSourceProviderProps
+};
+
+import { ICommonChartProps } from '../core/base/BaseChart';
+import { AreaChart as CoreAreaChart } from '../core/AreaChart';
+
+function generateDefaultDimensions(afm: AFM.IAfm): AFM.IDimension[] {
```

Comment:
this function seems to be the same for many charts, maybe we could extract and reuse